crates/sui-node/src/lib.rs

```diff
@@ -1,142 +1,125 @@
+use futures::TryFutureExt;
 pub use handle::SuiNodeHandle;
 use mysten_metrics::{spawn_monitored_task, RegistryService};
 use mysten_network::server::ServerBuilder;
 use mysten_service::server_timing::server_timing_middleware;
-use narwhal_network::metrics::MetricsMakeCallbackHandler;
-use narwhal_network::metrics::{NetworkConnectionMetrics, NetworkMetrics};
-use sui_archival::reader::ArchiveReaderBalancer;
-use sui_archival::writer::ArchiveWriter;
-use sui_config::node::{DBCheckpointConfig, RunWithRange};
-use sui_config::node_config_metrics::NodeConfigMetrics;
-use sui_config::object_storage_config::{ObjectStoreConfig, ObjectStoreType};
-use sui_config::{ConsensusConfig, NodeConfig};
-use sui_core::authority::authority_per_epoch_store::AuthorityPerEpochStore;
-use sui_core::authority::authority_store_tables::AuthorityPerpetualTables;
-use sui_core::authority::epoch_start_configuration::EpochStartConfigTrait;
-use sui_core::authority::epoch_start_configuration::EpochStartConfiguration;
-use sui_core::authority_aggregator::{AuthAggMetrics, AuthorityAggregator};
-use sui_core::authority_server::{ValidatorService, ValidatorServiceMetrics};
-use sui_core::checkpoints::checkpoint_executor::metrics::CheckpointExecutorMetrics;
-use sui_core::checkpoints::checkpoint_executor::{CheckpointExecutor, StopReason};
-use sui_core::checkpoints::{
-    CheckpointMetrics, CheckpointService, CheckpointStore, SendCheckpointToStateSync,
-    SubmitCheckpointToConsensus,
-};
-use sui_core::consensus_adapter::{
-    CheckConnection, ConnectionMonitorStatus, ConsensusAdapter, ConsensusAdapterMetrics,
-};
-use sui_core::consensus_manager::{ConsensusManager, ConsensusManagerTrait};
-use sui_core::consensus_throughput_calculator::{
-    ConsensusThroughputCalculator, ConsensusThroughputProfiler, ThroughputProfileRanges,
+use prometheus::Registry;
+use sui_archival::{reader::ArchiveReaderBalancer, writer::ArchiveWriter};
+use sui_config::{
+    node::{DBCheckpointConfig, RunWithRange},
+    node_config_metrics::NodeConfigMetrics,
+    object_storage_config::{ObjectStoreConfig, ObjectStoreType},
+    ConsensusConfig, NodeConfig,
 };
-use sui_core::consensus_validator::{SuiTxValidator, SuiTxValidatorMetrics};
-use sui_core::db_checkpoint_handler::DBCheckpointHandler;
-use sui_core::epoch::committee_store::CommitteeStore;
-use sui_core::epoch::consensus_store_pruner::ConsensusStorePruner;
-use sui_core::epoch::epoch_metrics::EpochMetrics;
-use sui_core::epoch::reconfiguration::ReconfigurationInitiator;
-use sui_core::module_cache_metrics::ResolverMetrics;
-use sui_core::overload_monitor::overload_monitor;
-use sui_core::rest_index::RestIndexStore;
-use sui_core::signature_verifier::SignatureVerifierMetrics;
-use sui_core::state_accumulator::StateAccumulator;
-use sui_core::storage::RocksDbStore;
-use sui_core::transaction_orchestrator::TransactiondOrchestrator;
 use sui_core::{
-    authority::{AuthorityState, AuthorityStore},
+    authority::{
```

Comment:
can we revert these formatting changes? There's really noisy and can result in bad merge conflicts